platforms/ios/demo/src/MapViewController.m

```diff
@@ -150,18 +165,32 @@ - (void)mapView:(TGMapView *)view recognizer:(UIGestureRecognizer *)recognizer d
     // Request feature picking
     [view pickFeatureAt:location];
     [view pickLabelAt:location];
-    // [view pickMarkerAt:location];
+    // [view pickMarkerAt:location];*/
 
     TGCameraPosition* camera = [view cameraPosition];
     camera.center = CLLocationCoordinate2DMake(coordinates.latitude, coordinates.longitude);
-    [view setCameraPosition:camera withDuration:0.5 easeType:TGEaseTypeCubic callback: ^(BOOL canceled){
+    [view setCameraPosition:camera withDuration:0 easeType:TGEaseTypeCubic callback: ^(BOOL canceled){
+        NSLog(@"Animation completed %d", !canceled);
+    }];
+}
+
+- (void)mapView:(TGMapView *)view recognizer:(UIGestureRecognizer *)recognizer didRecognizeDoubleTapGesture:(CGPoint)location {
+    CLLocationCoordinate2D coordinates = [view coordinateFromViewPosition:location];
+    TGCameraPosition* camera = [view cameraPosition];
+    camera.center = CLLocationCoordinate2DMake(coordinates.latitude, coordinates.longitude);
+    camera.zoom += 1;
+    [view setCameraPosition:camera withDuration:5000 easeType:TGEaseTypeCubic callback: ^(BOOL canceled){
```

Comment:
This animation is 5000 seconds long, probably not what you intended 😄

Reply:
Nopes, I purposely did 5000, its very soulful to take a slow route from sf to new delhi :P.

Reply:
Also this is from a drop commit.